charts/grafana/values.yaml

```diff
@@ -647,7 +648,7 @@ imageRenderer:
     portName: 'http'
     # image-renderer service port used by both service and deployment
     port: 8081
-    targetPort: 3001
+    targetPort: 8081
```

Comment:
It listens on port `8081` not `3001` so previously this just gave `Connection refused` errors